### PR TITLE
Remove commcare-hq src from formplayer (part 2)

### PR DESCRIPTION
**Pull request by dannyroberts**
Followup to https://github.com/dimagi/commcare-cloud/pull/4361. This is both just nice cleanup and something that we found since that PR was merged blocks us from running `update-config`. Once a formplayer deploy has gone out, current/localsettings.py will no longer exist and a task that tries to update it fails.

##### ENVIRONMENTS AFFECTED
all

### CODE REVIEW

File: src/commcare_cloud/ansible/deploy_commcarehq.yml [hunk 1]
```diff
@@ -73,22 +73,22 @@
   tags: services
 
 - name: Management Command Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_commcarehq.yml:76]: If the HQ code isn't on formplayer machines then we can't run management command from them either.

```suggestion
  hosts: commcarehq
```

File: src/commcare_cloud/ansible/deploy_commcarehq.yml [hunk 2]
```diff
@@ -73,22 +73,22 @@
...
   tasks:
     - include_vars: roles/commcarehq/vars/main.yml
     - include_tasks: roles/commcarehq/tasks/management_commands.yml
   tags: services
 
 - name: Prometheus Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_commcarehq.yml:83]: This is only needed if there are HQ processes running on the machine.
```suggestion
  hosts: commcarehq
```

File: src/commcare_cloud/ansible/deploy_commcarehq.yml [hunk 3]
```diff
@@ -73,22 +73,22 @@
...
   tasks:
     - include_vars: roles/commcarehq/vars/main.yml
     - include_tasks: roles/commcarehq/tasks/prometheus.yml
       when: prometheus_monitoring_enabled|default(False)
   tags: services
 
 - name: Prometheus Django runner
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```
Comment (snopoke) [inline src/commcare_cloud/ansible/deploy_commcarehq.yml:91]: Same as above:
```suggestion
  hosts: commcarehq
```

**Review by snopoke — CHANGES_REQUESTED** [review]

File: src/commcare_cloud/ansible/deploy_commcarehq.yml (hunk 1 above)
Comment (millerdev) [reply src/commcare_cloud/ansible/deploy_commcarehq.yml:76]: What management commands do we need to run on a formplayer machine?

Comment (snopoke) [reply src/commcare_cloud/ansible/deploy_commcarehq.yml:76]: @millerdev none, hence why I suggested removing it.

Comment (millerdev) [reply src/commcare_cloud/ansible/deploy_commcarehq.yml:76]: I'm sorry, I misinterpreted your comment. I thought you were saying we need to keep the commcarehq source on these machines so we can run management commands on them.

**Review by dannyroberts — COMMENTED** [review]
Thanks @snopoke for your careful review there. That makes this PR even simpler!

**Review by millerdev — APPROVED** [review]

**Pull request merged**